python/tvm/testing/utils.py

```diff
@@ -0,0 +1,1290 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=invalid-name,unnecessary-comprehension
+""" TVM testing utilities
```

Comment:
BTW, are the contents moved from anywhere else?

Reply:
yes the `python/tvm/testing.py` is renamed and moved here. But I kept the old `testing.py` and didn't change the references to it as I think that piece of work should belong to a new pr.

Reply:
Okay let's make sure there is a new PR coming soon that removes those copy-paste.